README.md

```diff
@@ -137,6 +137,7 @@ Search links that point directly to suitable issues to contribute to on GitHub.
 
 ## Using Version Control
 
+- [Video tutorial for Git and Github by Harvard University](https://www.youtube.com/watch?v=NcoBAfJ6l2Q&t=1604s&ab_channel=CS50) -  tutorial by Harvard University, part of their CS50 Web Development course on understanding Git and GitHub and working with Git commands.
```

Comment:
This link doesn't start the video at the beginning. Please replace it with this one that does.

```suggestion
- [Video tutorial for Git and Github by Harvard University](https://www.youtube.com/watch?v=NcoBAfJ6l2Q) -  tutorial by Harvard University, part of their CS50 Web Development course on understanding Git and GitHub and working with Git commands.
```